app/src/main/java/uk/gov/register/FieldsConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.register;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.google.common.collect.Lists;
+import io.dropwizard.jackson.Jackson;
+
+import java.io.*;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class FieldsConfiguration {
+
+    private final List<Field> fields;
+
+    public FieldsConfiguration(Optional<String> fieldsResourceYamlPath) {
```

Comment:
did you consider just reading from a fixed classpath location, and configuring at runtime by setting an appropriate classpath? how would that compare?